src/redux/books/books.jsx

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const bookSlice = createSlice({
+  name: 'books',
+  initialState: [],
+  reducers: {
+    bookAdded(state, action) {
+      state.push(action.payload);
+    },
+  },
+});
+
+export const selectBooks = (state) => state.books;
+export const { bookAdded } = bookSlice.actions;
```

Comment:
- [ ] Good job so far. You didn't implement the action to `REMOVE` the book because we are not able to remove a book. Kindly implement that action :+1:

Reply:
@andersonlebon, Thank you for reviewing my PR. I will be implementing all the changes requested.

Reply:
As you have agreed with the previous reviewer, kindly implement the `REMOVE` book functionality. This is part of the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/5801583dd2ad132a9326d4bd429291d954e4e824/bookstore/add_reducers_an_actions.md?plain=1#L43).